<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CReactionWidget</class>
 <widget class="QWidget" name="CReactionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CReactionEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CQtList" name="listReactants">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>â†’</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="CQtList" name="listProducts">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutReactants">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveReactant">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove selected reactant</string>
       </property>
       <property name="statusTip">
        <string>Remove selected reactant</string>
       </property>
       <property name="whatsThis">
        <string>Remove selected reactant</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddReactant">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add reactant</string>
       </property>
       <property name="statusTip">
        <string>Add reactant</string>
       </property>
       <property name="whatsThis">
        <string>Add reactant</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayoutProducts">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveProduct">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove selected product</string>
       </property>
       <property name="statusTip">
        <string>Remove selected product</string>
       </property>
       <property name="whatsThis">
        <string>Remove selected product</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddProduct">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add product</string>
       </property>
       <property name="statusTip">
        <string>Add product</string>
       </property>
       <property name="whatsThis">
        <string>Add product</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtList</class>
   <extends>QListWidget</extends>
   <header>QtList.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
